apps/server/src/modules/board/uc/media-board/media-board.uc.ts

```diff
@@ -0,0 +1,62 @@
+import { AuthorizationContextBuilder, AuthorizationService } from '@modules/authorization';
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { FeatureDisabledLoggableException } from '@shared/common/loggable-exception';
+import { BoardExternalReferenceType, type MediaBoard, type MediaLine } from '@shared/domain/domainobject';
+import type { User as UserEntity } from '@shared/domain/entity';
+import type { EntityId } from '@shared/domain/types';
+import type { MediaBoardConfig } from '../../media-board.config';
+import { MediaBoardService, MediaLineService } from '../../service';
+
+@Injectable()
+export class MediaBoardUc {
+	constructor(
+		private readonly authorizationService: AuthorizationService,
+		private readonly mediaBoardService: MediaBoardService,
+		private readonly mediaLineService: MediaLineService,
+		private readonly configService: ConfigService<MediaBoardConfig, true>
+	) {}
+
+	public async getMediaBoardForUser(userId: EntityId): Promise<MediaBoard> {
+		this.checkFeatureEnabled();
+
+		const user: UserEntity = await this.authorizationService.getUserWithPermissions(userId);
+		this.authorizationService.checkPermission(user, user, AuthorizationContextBuilder.read([]));
+
+		const boardIds: EntityId[] = await this.mediaBoardService.findIdsByExternalReference({
+			type: BoardExternalReferenceType.User,
+			id: user.id,
+		});
+
+		let board: MediaBoard;
+		if (!boardIds.length) {
```

Comment:
Do we want to do anything like looks if the a user accidentally have more than one boards or when a new board will be created?

Reply:
I think this is a question for the business logic ticket